app/models/chargeback.rb

```diff
@@ -61,7 +61,7 @@ def self.build_results_for_report_chargeback(options)
         # we need to select ChargebackRates for groups of MetricRollups records
         # and rates are selected by first MetricRollup record
         metric_rollup_record = metric_rollup_records.first
-        rates_to_apply = cb.get_rates(metric_rollup_record)
+        rates_to_apply = cb.uniq_rates(metric_rollup_record, options.try(:tag))
```

Comment:
I think we don't need `.try(:` here.

This comment however is nitpick and can be ignored.